### PR TITLE
Add Bcftools stats and change VCF compression

### PR DESCRIPTION
**Pull request by jfy133**
This adds bcftools stats for #317 and also at request of @TCLamnidis for more standard compression format for VCFtools.

TODO:

- [x] Output docs (multiqc + output files
- [x] Check bzgip input works for MultiVCFAnalyzer and VCF2GEnome, and if modify theses processes to accept them
- [x] Add software versions!

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
  - [ ] If you've added a new tool - add to the software_versions process and a regex to `scrape_software_versions.py`
  - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/eager/tree/master/.github/CONTRIBUTING.md)
  - [x] If necessary, also make a PR on the nf-core/eager _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint .`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker`).
- [x] Usage Documentation in `docs/usage.md` is updated.
- [x] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [x] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: docs/images/nf-core_eager_logo_small.svg [hunk 1]
```diff
@@ -0,0 +1,503 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (apeltzer) [inline docs/images/nf-core_eager_logo_small.svg:1]: Why did this file change at all?

File: docs/output.md [hunk 2]
```diff
@@ -618,6 +618,32 @@ If this correlation is not observed, your data is skewed towards higher coverage
   <img src="images/output/sexdeterrmine/sexdeterrmine_read_counts.png" width="75%" height = "75%">
 </p>
 
+### Bcftools
+
+### Background
+
+Bcftools is a toolkit for processing and summaries VCF files, i.e. variant call format files. nf-core/eager currently uses bcftools for the `stats` functionality. This summarises in a text file a range of statistics about VCF files, produced by GATK and FreeBayes variant callers.
```
Comment (apeltzer) [inline docs/output.md:625]:
```suggestion
Bcftools is a toolkit for processing and summarising of VCF files, i.e. variant call format files. nf-core/eager currently uses bcftools for the `stats` functionality. This summarises in a text file a range of statistics about VCF files, produced by GATK and FreeBayes variant callers.
```

File: main.nf [hunk 3]
```diff
@@ -2213,8 +2213,24 @@ if ( params.run_genotyping && params.genotyping_source == 'raw' ) {
...
+//  ch_fb_for_bcftools_stats = Channel.empty()
+//} else if ( params.genotyping_tool == 'hc' ) {
+//  ch_ug_for_bcftools_stats = Channel.empty()
+//  ch_fb_for_bcftools_stats = Channel.empty()
+//} else if ( params.genotyping_tool == 'fb ') {
+//  ch_ug_for_bcftools_stats = Channel.empty()
+//  ch_hc_for_bcftools_stats = Channel.empty()
+//}
+
+
```
Comment (apeltzer) [inline main.nf:2233]:
```suggestion
```


Drop this entirely?

File: main.nf [hunk 4]
```diff
@@ -2473,9 +2514,9 @@ process vcf2genome {
   def out = !params.vcf2genome_outfile ? "${samplename}.fasta" : "${params.vcf2genome_outfile}"
   def fasta_head = !params.vcf2genome_header ? "${samplename}" : "${params.vcf2genome_header}"
   """
-  pigz -f -d -p ${task.cpus} *.vcf.gz
-  vcf2genome -Xmx${task.memory.toGiga()}g -draft ${out}.fasta -draftname "${fasta_head}" -in ${vcf.baseName} -minc ${params.vcf2genome_minc} -minfreq ${params.vcf2genome_minfreq} -minq ${params.vcf2genome_minq} -ref ${fasta} -refMod ${out}_refmod.fasta -uncertain ${out}_uncertainy.fasta
-  pigz -p ${task.cpus} *.fasta 
+  pigz -d -f -p ${task.cpus} ${vcf}
```
Comment (apeltzer) [inline main.nf:2517]: Above its bgzip, here its pigz? Should we make all bgzip just to be safe?

**Review by apeltzer — APPROVED** [review]
I think ok, but some minor (cosmetic) things

File: docs/images/nf-core_eager_logo_small.svg (hunk 1 above)
Comment (jfy133) [reply docs/images/nf-core_eager_logo_small.svg:1]: No clue! But still loads for me!

**Pull request merged**